src/services/template/UserResolvers.ts

```diff
@@ -131,4 +134,73 @@ export class UserResolver {
 
         return true;
     }
+
+    @Mutation(() => Boolean)
+    async resetPasswordToken(
+        @Arg("email") email: string
+    ) {
+        const buf = crypto.randomBytes(20)
+        const token = buf.toString('hex');
+
+        try {
+            const user = await User.findOne({ email: email });
+            user.resetPasswordToken = token;
+            user.resetPasswordExpires = 3600000; // 1 hour
```

Comment:
`user.resetPasswordExpires = new Date() + 3600000;`

This will give you the timestamp of then the token expires, so you will actually be able to tell when it does expire